utils/nitro_utils.h

```diff
@@ -246,7 +246,7 @@ inline void nitro_logo() {
 }
 
 inline drogon::HttpResponsePtr nitroHttpResponse() {
-  auto resp = drogon::HttpResponse::newHttpResponse();
+  auto resp = drogon::HttpResponse::newHttpResponse(drogon::k200OK, drogon::CT_APPLICATION_JSON);
```

Comment:
Referring to https://github.com/drogonframework/drogon/blob/96919df488e0ebaa0ed304bbd76bba33508df3cc/lib/src/HttpUtils.cc#L43